docker/dev.yml

```diff
@@ -34,7 +47,7 @@ services:
         USER_UID: ${USER_UID:-1000}
         USER_GID: ${USER_GID:-1000}
     environment:
-      - REACT_APP_API_URL=//api.${TAXONOMY_EDITOR_DOMAIN}:${TAXONOMY_EDITOR_PORT:-80}/
+      - REACT_APP_API_URL=http://api.${TAXONOMY_EDITOR_DOMAIN}${PUBLIC_TAXONOMY_EDITOR_PORT:-80}/
```

Comment:
don't add the http because in production we use https.

If you use "://" it uses the current page scheme.